server/market/market.go

```diff
@@ -74,6 +74,10 @@ type FeeFetcher interface {
 	MaxFeeRate() uint64
 }
 
+type Balancer interface {
```

Comment:
Doc or maybe even unexport `Balancer`?  It's internal to the `server/market` package anyway, only an interface because a concrete `DEXBalancer` is a headache to create, right?

Reply:
Oh, no it's used in the NewMarket constructor from server/dex.... OK, doc I guess